barustenberg/src/plonk/proof_system/verifier/test.rs

```diff
@@ -1,6 +1,16 @@
+use ark_bn254::Fq12;
```

Comment:
this blocks using other curves (probably ok here but not in evaluationdomain)

Reply:
yes